Jenkinsfile

```diff
@@ -155,6 +162,16 @@ spec:
     tty: true
     command:
     - 'cat'
+  - name: 'eksctl'
+    image: weaveworks/eksctl
```

Comment:
We should be specifying the docker image tag

Reply:
Done.